Patches.py

```diff
@@ -1376,14 +1376,10 @@ def set_entrance_updates(entrances):
         save_context.write_bits(0x00D4 + 0x0C * 0x1C + 0x04 + 0x3, 0xDC) # Thieves' Hideout switch flags (heard yells/unlocked doors)
         save_context.write_bits(0x00D4 + 0x0C * 0x1C + 0x0C + 0x2, 0xC4) # Thieves' Hideout collection flags (picked up keys, marks fights finished as well)
 
-    # Add a gate-opening guard on the Wasteland side of the Gerudo gate when the card is shuffled or certain levels of ER.
-    # Overrides the generic guard at the bottom of the ladder in Gerudo Fortress
```

Comment:
I think this comment should be kept since it's still relevant and clarifies which actor it overrides

Reply:
Good call, added the second half of the comment back